lib/glific/clients/digital_green.ex

```diff
@@ -173,6 +173,35 @@ defmodule Glific.Clients.DigitalGreen do
       else: %{is_valid: false}
   end
 
+  def webhook("push_crop_calendar_message", fields) do
+    crop_age = fields["crop_age"]
+
+    {:ok, organization_data} =
+      Repo.fetch_by(OrganizationData, %{
+        organization_id: fields["organization_id"],
+        key: fields["crop"]
+      })
+
+    template_uuid = get_in(organization_data.json, [crop_age, "template_uuid"])
+    variables = get_in(organization_data.json, [crop_age, "variables"])
+    crop_stage = get_in(organization_data.json, [crop_age, "crop_stage"])
+    media_url = get_in(organization_data.json, [crop_age, "media_url"])
+    crop_stage_eng = get_in(organization_data.json, [crop_age, "crop_stage_eng"])
```

Comment:
@mohitgusain this will not work as the key should be either string or tuple
`get_in(organization_data.json, [:crop_age, "template_uuid"])` or 
`get_in(organization_data.json, ["crop_age", "template_uuid"])`

Reply:
@AkhileshNegi I don't think organization_data.json can hold an atom (becuase it's a json not an enum) so are you sure this will work ?

Reply:
Also I think crop_age is a variable so it should work. Am I missing something 🤔 ?

Reply:
@pankaj-ag I was talking about the get_in function so the correct way would be `get_in(organization_data, [:json, "crop_age", "template_uuid"])`